Detectors/TRD/reconstruction/include/TRDReconstruction/CruRawReader.h

```diff
@@ -115,18 +109,24 @@ class CruRawReader
   // - whenever we don't know the half-chamber, fill it for HCID == 1080 so that we can still disentangle errors from half-chamber 0?
   // - probably enough to only pass the half-chamber ID, if we know it
   // - or what about being more granular? dump ROB or MCM, if we know it?
-  void incrementErrors(int hist, int sector = -1, int side = 0, int stack = 0, int layer = 0);
+  void incrementErrors(int error, int hcid = -1);
 
   // helper function to dump the whole input payload including RDH headers
   void dumpInputPayload() const;
 
+  // dump out and entire half cru buffer.
+  void OutputHalfCruRawData(uint32_t offset);
```

Comment:
A tiny thing, but still... Class methods in O2 should start with lower case letter...

Reply:
fixed